vault/resource_kubernetes_auth_backend_config_test.go

```diff
@@ -84,6 +85,8 @@ func TestAccKubernetesAuthBackendConfig_import(t *testing.T) {
 						"pem_keys.0", kubernetesPEMfile),
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_config.config",
 						"pem_keys.#", "1"),
+					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_config.config",
+						"issuer", issuer),
```

Comment:
It would be great to get an "issuer not included still behaves the same way as before" test here too.

Reply:
PTAL https://github.com/terraform-providers/terraform-provider-vault/pull/601/commits/6b5077f7f4b3241c0b1f8acf6e3532d14d9712f3